OutOfSchool/OutOfSchool.WebApi/Common/Responses/InputApiError.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OutOfSchool.WebApi.Common.Responses;
+
+public class InputApiError : ApiError
+{
+    public static readonly InputApiError InputDataIncorrect
+        = new (nameof(InputDataIncorrect), "Provided data input is incorrect");
```

Comment:
(Non-uniqueness) `nameof` is a cool idea, but it will give you only method name. Thus, in other class (i.e. `InputAdminApiError`) method with same name (`InputDataIncorrect`) could exist and you'll get eror-code collision. Please use simple enum containing all error codes, or more complex schema ot generate them - at least 
`$"{nameof(class)}.{nameof(method}"`